devices/microsoft/research/qr-url-device-map.csv

```diff
@@ -0,0 +1,27 @@
+vanity name, design id, revision, module name, redirect source
```

Comment:
did you reserve those? IMO, we shoud start with "jd*"

Reply:
They are all reserved.

Reply:
Feel free to go reserve some more! The assigned ones have shipped